test/controllers/update/update.trusts.submit.by.paper.controller.spec.ts

```diff
@@ -57,12 +57,5 @@ describe("Update trusts submit by paper controller", () => {
       expect(resp.text).toContain(UPDATE_TRUSTS_SUBMIT_BY_PAPER_PAGE_HEADING);
       expect(resp.text).toContain(UPDATE_OVERSEAS_ENTITY_CONFIRM_URL);
     });
-
```

Comment:
looks like this test can still be used as it is testing when FEATURE_FLAG_ENABLE_UPDATE_TRUSTS = true

Reply:
Again not what this test is for and whether we should be expecting 404 it is very unclear as to why.

Reply:
Are we clear to remove submit by paper completely?

Reply:
That needs clarifying by the analyst and if so, should be done under another ticket. There is a controller, a web page and routing that would need to be removed which is beyond the scope of this ticket.

Reply:
I think we can dispense with the test otherwise it will just fail if we try to bring it back in.